language/bytecode-verifier/bytecode-verifier-tests/src/unit_tests/signature_tests.rs

```diff
@@ -122,190 +122,3 @@ fn no_verify_locals_good() {
     };
     assert!(VerifiedModule::new(compiled_module_good.freeze().unwrap()).is_ok());
 }
-
-#[test]
-fn no_verify_locals_bad1() {
```

Comment:
What are these tests exactly and why are they gone?

Reply:
These are checking if the parameters are a prefix of the locals and with this change they are no longer needed.